### DISCUSSION TITLE
Banking software development company

### DISCUSSION THREAD

**Discussion by vistonviston** [community/community#132545] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi all! I'm searching for a banking software development company. I want to order a fintech app, it ought to be easy to use, adaptable for future improvements, and compliant with industry standards. The budget is negotiable. 
Thanks in advance

**Comment by ebndev** [comment]
Hi @vistonviston, thanks for being a part of the GitHub Community!

Please note: we currently do not allow job listings or solicitation on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍

**Comment by blackbanj** [comment]
Hello!
I can’t recommend you any specific company, but I can advise you to read this article [https://itexus.com/top-10-banking-software-development-companies/](https://itexus.com/top-10-banking-software-development-companies/). I think, by reading it, you can make a more informed decision and choose a banking software development company that will meet your needs and help achieve your business goals.

**Reply by vistonviston** [reply]
> Hello! I can’t recommend you any specific company, but I can advise you to read this article https://itexus.com/top-10-banking-software-development-companies/. I think, by reading it, you can make a more informed decision and choose a banking software development company that will meet your needs and help achieve your business goals.

Thank you for sharing!

**Comment by fabianrod** [comment]
I'd like to recommend sundevs.com, they have experience building for some big companies and they can offer  [low code development services](https://www.sundevs.com/low-code), it's a better way to save money when you hire a software company.

**Reply by vistonviston** [reply]
Thank you!

**Comment by vistonviston** [comment]
Guys, thank you all for your answers. I asked this question on several forums and chose Softellar. Here is their [website](https://softellar.com/). At the moment, I liked them the most out of the three I considered. They have the best conditions and reviews. What do you think about this team? Maybe someone has already worked with them? Are there any nuances of work? Or is everything ok?
